README.md

```
@@ -5,7 +5,33 @@
 [![Docs](https://img.shields.io/badge/docs-readthedocs-blue)](https://agentic-slayer.readthedocs.io/)
 [![License](https://img.shields.io/github/license/MotleyAI/slayer)](LICENSE)
 
-A lightweight open-source semantic layer for AI agents and humans
+A lightweight, open-source semantic layer that lets AI agents query data without writing SQL.
+
+---
+
+### The problem
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)